ci-operator/step-registry/opendatahub-io/ci/s390x-build/create/opendatahub-io-ci-s390x-build-create-commands.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+set -x
```

Comment:
We'll need to run this at first to make sure it's working, don't wanna integrate thiss and have all failing builds

Reply:
(Second round review) we can test it as we go, I asked and I'm not sure we can execute scripts in openshift ci before submitting

Reply:
This is a leap of faith to me, I mostly understand the script but I cannot test it, we can verify this after submitted